adapters/verizonmedia/verizonmedia.go

```diff
@@ -0,0 +1,189 @@
+package verizonmedia
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+)
+
+type VerizonMediaAdapter struct {
+	http *adapters.HTTPAdapter
+	URI  string
+}
+
+func (a *VerizonMediaAdapter) Name() string {
+	return "verizonmedia"
+}
+
+func (a *VerizonMediaAdapter) SkipNoCookies() bool {
+	return false
+}
+
+func (a *VerizonMediaAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+
+	errs := make([]error, 0, len(request.Imp))
+	if len(request.Imp) == 0 {
+		err := &errortypes.BadInput{
+			Message: "No impression in the bid request",
+		}
+		errs = append(errs, err)
+		return nil, errs
+	}
+
+	errors := make([]error, 0, 1)
+
+	var bidderExt adapters.ExtImpBidder
+	err := json.Unmarshal(request.Imp[0].Ext, &bidderExt)
```

Comment:
Are we only concerned about the first `Imp`? Other adapters are "multi-imp" and loop through the `Imp` array to verify for relevant `Imp`s. Is that relevant in our context?
```
 26 func (a *VerizonMediaAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.R>
 27 +-- 12 lines: errs := make([]error, 0, len(request.Imp))---------
 39     var bidderExt adapters.ExtImpBidder
 40     err := json.Unmarshal(request.Imp[0].Ext, &bidderExt)
 41     if err != nil {
 42         err = &errortypes.BadInput{
 43             Message: "ext.bidder not provided",
 44         }
 45         errors = append(errors, err)
 46         return nil, errors
 47     }
 48     var verizonMediaExt openrtb_ext.ExtImpVerizonMedia
 49     err = json.Unmarshal(bidderExt.Bidder, &verizonMediaExt)
 50     if err != nil {
```

Reply:
We are concerned only about the first entry. The back-end supports only 1 entry , it processes only the first imp of the array. I've asked for a double confirmation on this.

Reply:
I got the confirmation again that the back-end only uses the first entry of the Imp array, it supports only 1 Imp per BidRequest